code/reviewers/review_cleaner.do

```diff
@@ -247,12 +247,14 @@ label values race_res race_res
 la var race_res "Race"
 la var sex_res "Sex"
 
-drop if sex_res > 2
```

Comment:
I dropped all combinations of sex/race that weren't male female and A/H/B/W. I see you're removing all the drop commands. Is there a reason?

Reply:
I removed it for two reasons: 1) we were trying to recreate the same number of observations as those included in your original table. This did the trick. 2) I included the "sample" variable so that we can always restrict the data to the correct specifications, without dropping parts of the full data.